libosdp/src/commands.rs

```diff
@@ -7,7 +7,7 @@
 //! are specified by OSDP specification. This module is responsible to handling
 //! such commands though [`OsdpCommand`].
 
-use crate::OsdpStatusReport;
+use crate::{OsdpStatusReport, Vec};
```

Comment:
Here and other places, isn't it better to make the import location more obvious than `crate::`? To me it looks like these types are being provided by this crate.

Since I'm very new to Rust, I don't know if this is an accepted convention -- in which case we can leave it as is. If not, I suggest doing something like:

```suggestion
use crate::OsdpStatusReport;
use alloc::vec::Vec;
```

Reply:
I agree. That is not the common Rust convention and possibly confusing.